docs/mobile/development.md

```diff
@@ -102,25 +102,73 @@ mobile device.
 
 ## Release
 
-1. Merge your content branch into the *development* branch on GitHub.
+### Development App
 
-2. Check that the build passes on
-   [CircleCI](https://circleci.com/gh/datacamp/workflows). This should take a
-   few minutes or less.
+At any point, you can preview your content on the development version of the
+DataCamp for Mobile app. This is a second app that you can install alongside the
+real production version of the app. It displays the content that you push to the
+`development` (dev) branch. Only you and DataCamp will be able to see what you
+push to dev. Email spencer@datacamp.com for an invite to the iOS/Android dev
+app.
 
-3. Download the latest development build of DataCamp for Mobile.
+When you want to push to dev:
 
-   * [Android App
-     Center](https://appcenter.ms/users/mobileDCamp/apps/DataCamp-Android/distribute/distribution-groups/Collaborators/releases)
-   * [iTunes TestFlight](https://itunesconnect.apple.com/WebObjects/iTunesConnect.woa/ra/ng/app/1321970315/testflight?section=iosbuilds)
+1. First, make sure everything is working:
 
-4. Check out your new content on the development app, make sure everything looks
+```sh
+mobile-teach check
+```
+
+2. Reset the `development` branch to the content you want to test.
+
+```sh
+## Checkout the development branch
+git checkout -b development
+
+## Reset the development branch to your content
+git reset --hard chapter-2
+
+## Force push development
+git push origin development --force
```

Comment:
Best to use `--force-with-lease`, very similar but a bit more safe: 

`git push origin development --force-with-lease`